src/common/src/config.rs

```diff
@@ -741,6 +747,10 @@ pub mod default {
             30
         }
 
+        pub fn vacuum_spin_interval_ms() -> u64 {
```

Comment:
Is the default value of vacuum_spin_interval_ms too large?  Is there a reference for this value?

Reply:
Reduced to 10ms.